classes/02class/exercises/c02-network02/rodrigovcesar/README.md

```diff
@@ -0,0 +1,25 @@
+# c02-network02
+
+## Commands Execution Output
+
+- Answers for `10.0.0.0/16`:
+
+|Subnet|Network|Host Range|Usable Hosts|Broadcast|AWS Reserved|
+|-|-|-|-|-|-|
+|subnet-a| 10.0.0.0/18 | 10.0.0.0 - 10.0.63.255 | 10.0.0.4 - 10.0.63.254| 10.0.63.255 | 10.0.0.1, 10.0.0.2, 10.0.0.3
+|subnet-b| 10.0.64.0/18 | 10.0.64.0 - 10.0.127.255 | 10.0.64.4 - 10.0.127.254 | 10.0.127.255| 10.0.64.1, 10.0.64.2, 10.0.64.3
+|subnet-c| 10.0.128.0/18 | 10.0.128.0 - 10.0.191.255 | 10.0.128.4 - 10.0.191.254 | 10.0.191.255 | 10.0.128.1, 10.0.128.2, 10.0.128.3
+|subnet-d| 10.0.192.0/18 | 10.0.192.0 - 10.0.255.255| 10.0.192.4 - 10.0.255.254 | 10.0.255.255 | 10.0.192.1, 10.0.192.2, 10.0.192.3 
+
+- Answers for `192.168.0.0/24`:
+
+|Subnet|Network|Host Range|Usable Hosts|Broadcast|AWS Reserved|
+|-|-|-|-|-|-|
+|subnet-a| 192.168.0.0/26 | 192.168.0.0 - 192.168.0.63 | 192.168.0.4 - 192.168.0.62 | 192.168.0.63 | 192.168.0.1, 192.168.0.2. 192.168.0.63
+|subnet-b| 192.168.0.64/26 | 192.168.0.64 - 192.168.0.127| 192.168.0.65 - 192.168.0.126 | 192.168.0.127 | 192.168.0.65, 192.168.0.66, 192.168.0.67
```

Comment:
Hey Rodrigo, great work on the subnetting!

Technically you are not wrong here (I think AWS reserved IPs can also count as usable hosts) but since your other answers don't include AWS reverse IPs on usable host, maybe you can change it accordingly to maintain the consistency? 😄

Reply:
Sorry, I didn't see it before. Yep, I will change it.

Reply:
I made the corrections, I think I was hurry. I found an error also on line 18 hehehehe